lib/commands/config/generate.ts

```diff
@@ -26,26 +27,6 @@ import {
 } from '../../utils/messages';
 import type { BalenaSDK, PineDeferred } from 'balena-sdk';
 
-interface FlagsDef {
```

Comment:
@thgreasi @myarmolinsky This is one example of one of the big wins in this new version. the static "flags" types now are inffered from what you added on your flags so you get type checking working fine on your flags ("options") without manually needing to cast to what are the types of the flags